.github/workflows/windows.yml

```diff
@@ -44,13 +42,8 @@ jobs:
           powershell ./infomaniak-build-tools/windows/build-drive.ps1 -ci
         shell: cmd
 
-      - name: Execute common tests
-        run: ./kDrive_test_common.exe
-        working-directory: ${{env.bin-directory}}
-
-      - name: Execute parms tests
-        run: ./kDrive_test_parms.exe
-        working-directory: ${{env.bin-directory}}
```

Comment:
No need for granting execute permission?

Reply:
Not needed for Windows